### PR TITLE
Feature: Services Section

### PR DESCRIPTION
**Pull request by jayhogan-dev**
- reusable service card component
- on hover, card moves down as shown in figma
- shadows added
- mobile responsive and dark mode enabled
image
image
image

### CODE REVIEW

File: components/ui/card.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import * as React from "react"
```
Comment (Nassim-Saboundji) [inline components/ui/card.tsx:1]: Can't you simply do `import React from "react"`?

File: components/ui/card.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+import { cn } from "@/lib/utils"
+
+const Card = React.forwardRef<
```
Comment (Nassim-Saboundji) [inline components/ui/card.tsx:5]: Would you mind explaining why you need forwardRefs?

**Review by Nassim-Saboundji — APPROVED** [review]
Looks good!

Just have a few questions.

File: components/ui/card.tsx (hunk 2 above)
Comment (jayhogan-dev) [reply components/ui/card.tsx:5]: This is from shadcn

File: components/ui/card.tsx (hunk 1 above)
Comment (jayhogan-dev) [reply components/ui/card.tsx:1]: This is also from shadcn

**Pull request merged**